espresso-support/core/build.gradle

```diff
@@ -3,33 +3,34 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath librariesBuild.androidGradleBuildTools
-        classpath librariesBuild.bintrayRelease
+        classpath libraries.build.androidGradle
+        classpath libraries.build.bintrayRelease
     }
 }
 
 apply plugin: 'com.android.library'
 apply plugin: 'bintray-release'
 
 android {
-    compileSdkVersion androidCompileSdkVersion
-    buildToolsVersion androidBuildToolsVersion
+    compileSdkVersion versions.androidSdk.compile
+    buildToolsVersion versions.androidSdk.buildTools
 
     defaultConfig {
-        minSdkVersion androidMinSdkVersion
+        minSdkVersion versions.androidSdk.min
     }
 }
 
 dependencies {
-    compile librariesTest.androidTestRunner
-    compile librariesTest.androidEspressoCore
+    provided project(':extras')
```

Comment:
`extras` includes `ViewActivity`.

This means clients can add the `extras` artifact as `debugCompile/espressoCompile` as part of their app under test, so the activity is registered by the package manager.

The `core` artifact just needs to know about `ViewActivity` for compilation. If the client is not using `ViewActivityRule` then they don't need to include the `extras` artifact, and can use the `core` artifact.

of course, the core artifact only includes `ViewActivityRule` right now so might be useful 😁  but we can also add common espresso things in here like custom gestures.